### DISCUSSION TITLE
Working with React and TalkJS

### DISCUSSION THREAD

**Discussion by bestassist112** [Real-Time-Chat/react-chat-app#2] [Q&A] [answered]
Most of the TalkJS code will live in the frontend, so it seamlessly integrates with React. Using TalkJS, you can turn any <div> into a full-fledged messaging app and have it up and running in a couple of hours.

**Comment by VitaminC1211** [comment, marked as answer]
Yes, correct, and also TalkJS allows you to ship faster, simplify your stack, and save on development costs. You get the backend and frontend, all you need to add a full-fledged chat into your React app.

**Comment by superstar1205** [comment]
Javascript SDK
Connect your users with 1-on-1 chat, group chat, or chatrooms using our real-time messaging infrastructure and pre-built UI.
